model/clusters_mgmt/v1/resource_type.model

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// A Resource which belongs to a cluster, example Cluster Deployment.
+class Resource {
+  // Type of resource
+  Type String
+	// Content of the resource.
+	Content String
```

Comment:
Indentation is off. Also, why is `Content` a string? Does the API return it all as a serialized string or should it be an `interface{}` type to allow for complex nested structures?

Reply:
Yeah, content is a serialized string, not sure why we done it that way, but looks similar to what is returned via `logs`. 
@tzvatot @nimrodshn should we change what is returned?